script/test/fixtures/volume-mounts/volumes-from/docker-compose.yml

```diff
@@ -17,4 +17,4 @@ services:
     volumes:
       - /www/public
     volumes_from:
-      - web
+      - web
```

Comment:
you've got to do `git checkout script/test/fixtures/volume-mounts/volumes-from/docker-compose.yml` to revert it back to normal. you've modified something here (newline maybe?)